airbyte-integrations/connectors/source-myhours/bootstrap.md

```diff
@@ -0,0 +1,13 @@
+## Streams
+
+This connector has the following streams, and all of them support full refresh only.
+
+* [Time Logs](https://documenter.getpostman.com/view/8879268/TVmV4YYU#a023832e-c39d-4cff-a639-d673fb8846c1)
+* [Clients](https://documenter.getpostman.com/view/8879268/TVmV4YYU#79916508-c2ba-4ed4-9d97-bbb769687c11)
+* [Projects](https://documenter.getpostman.com/view/8879268/TVmV4YYU#64fa3d61-a785-4727-bd33-f549b987c7b2)
+* [Tags](https://documenter.getpostman.com/view/8879268/TVmV4YYU#a7ef468e-120b-40de-ad52-79e9d485f688)
+* [Users](https://documenter.getpostman.com/view/8879268/TVmV4YYU#da5fa9cc-f337-4888-bf18-21e68a07ee3d)
+
+## Authentication
+
+My Hours exposes a REST API with username/password authorization. Giving the possibilities to use a refresh token aftwerwards. Since this refresh token is auto-rotating this connector only supports username/password.
```

Comment:
Since My Hours uses auto-rotating refresh tokens and #3990 will be needed to use this. Should I perhaps remove the this line to prevent this confusion?